rust-actix/src/frontend/static/style.css

```diff
@@ -0,0 +1,178 @@
+body {
```

Comment:
we need to do the global style.css and do a refactor in all projects for that. The sooner the better